Add `resetNashSolver()` to the public API

bzinberg opened the pull request:
Fixes https://github.com/JuliaPolyhedra/lrslib/issues/2.

## Tested
Builds successfully on my machine.

bzinberg committed:
Make `FirstTime` `extern`

See https://github.com/JuliaPolyhedra/lrslib/issues/2

giordano commented on lrsnashlib.h:
```diff
@@ -67,7 +67,7 @@ void initFwidth(game *g);
 
 
-long FirstTime;          /* set this to true for every new game to be solved */
+extern long FirstTime;   /* set this to true for every new game to be solved */
```
Why defining it here at all?

bzinberg replied:
My reading of the standard is that every variable must be defined exactly once, and an `extern` declaration is not a definition.

giordano replied:
So you can remove this line :slightly_smiling_face:

bzinberg replied:
then the variable would have internal linkage

blegat commented:
Can you redirect the change to the branch of https://github.com/JuliaPolyhedra/lrslib/pull/4 ? We won't be making change for the old version of lrslib, let's focus on v7.1a. There the definition in the `.h` was removed and we need to add it back, see https://github.com/JuliaPolyhedra/LRSLib.jl/pull/41#discussion_r618159018

blegat commented:
Probably even better is to add a function for modifying `FirstTime` and not defining the variable in `.h`.

bzinberg commented:
> Probably even better is to add a function for modifying `FirstTime` and not defining the variable in `.h`.

Responded to this in https://github.com/JuliaPolyhedra/LRSLib.jl/pull/41#discussion_r618515852

bzinberg commented:
> Can you redirect the change to the branch of #4 ? We won't be making change for the old version of lrslib, let's focus on v7.1a. There the definition in the `.h` was removed and we need to add it back, see [JuliaPolyhedra/LRSLib.jl#41 (comment)](https://github.com/JuliaPolyhedra/LRSLib.jl/pull/41#discussion_r618159018)

Absolutely.  Thanks for working on the migration!

bzinberg committed:
Instead of exposing `FirstTime` directly, expose a reset function

In preparation for rebasing this PR onto https://github.com/JuliaPolyhedra/lrslib/pull/4

This is a backward-incompatible change that will break earlier versions of LRSLib.jl.

bzinberg committed:
Merge branch 'v7.1' of github.com:JuliaPolyhedra/lrslib

bzinberg commented:
> Can you redirect the change to the branch of #4 ?

Done, and changed the target branch of this PR accordingly.

blegat reviewed (approved):
Thanks!
